app/src/main/java/org/liberty/android/fantastischmemo/dao/CardDaoImpl.java

```diff
@@ -431,6 +431,32 @@ public long getScheduledCardCount(Category filterCategory, Date startDate, Date
         }
     }
 
+    public long getNewLearnedCardCount(Category filterCategory, Date startDate, Date endDate) {
+        try {
+            LearningDataDao learningDataDao = getHelper().getLearningDataDao();
+            QueryBuilder<Card, Integer> cardQb = queryBuilder();
+            QueryBuilder<LearningData, Integer> learnQb = learningDataDao.queryBuilder();
+            cardQb.setCountOf(true);
+            cardQb.selectColumns("id");
+            learnQb.selectColumns("id");
+
+            learnQb.where().le("firstLearnDate", endDate)
+                    .and().ge("firstLearnDate", startDate)
+                    .and().gt("acqReps", "0").prepare();
+
+            Where<Card, Integer> where = cardQb.where().in("learningData_id", learnQb);
+            if (filterCategory != null) {
+                where.and().eq("category_id", filterCategory.getId());
+            }
+            cardQb.setWhere(where);
+
+            return countOf(cardQb.prepare());
+        } catch (SQLException e) {
+            e.printStackTrace();
```

Comment:
That's OK. I think you can delete it here. I can clean up the other places if they still print stack track. The problem is it does not work well with Android logcat.